src/ontology/foodon-edit.owl

```diff
@@ -18198,6 +18198,142 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001587"/>
+        <obo:IAO_0000115 xml:lang="en">A seed of an almond without skin.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-07-08T20:34:17Z</dc:date>
+        <rdfs:label xml:lang="en">almond meat</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003910 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003910">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003083"/>
+        <obo:IAO_0000115 xml:lang="en">A meat of a very young calf which is usually slaughtered as early as 2 hours or 2–3 days old (at most 1 month old).</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Veal</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-07-08T20:43:02Z</dc:date>
+        <rdfs:label xml:lang="en">bob veal</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003911 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003911">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003083"/>
+        <obo:IAO_0000115 xml:lang="en">A meat of a calf that is raised on a fortified milk formula diet plus solid feed.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Veal</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-07T20:53:21Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">formula-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">milk fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">milk-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">special fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">special-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">white veal</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">formula fed veal</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003912 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003912">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03450006"/>
+        <obo:IAO_0000115 xml:lang="en">A Japanese cooking technique in which food is broiled or grilled with a glaze of soy sauce, mirin, and sugar.</obo:IAO_0000115>
```

Comment:
Re. "A Japanese cooking technique ...". So I see Wikipedia offers up "teriaki" as both a cooking method and a noun. 
 Lets have a label "teriaki cooking method" and broad synonym "teriaki".  That leaves space to add a "teriaki dish" as well and eliminate ambiguity.  We can use this pattern going forward for all cooking methods (processes). Technique seems to be a combination of method and ingredients (the sauce) sometimes but I think its safe to equate with process.  Teriaki sauce can be an input to the sauce.

Reply:
I agree and it's a good idea to change the label and adding a synonym to it so we can add teriyaki dish as well. I will add that. But does equating it with the process here means using something like 'A Japanese cooking process..? which will include the ingredient (teriyaki sauce) in definition.

Reply:
Yes re "A japanese cooking method ..." - with the aim to keep the definition as succinct as possible and push off other details to rdfs:comment.  Seems it literally would be a food preparation process.

Reply:
Got it, thanks.